src/include/catalog/database_catalog.h

```diff
@@ -36,6 +36,11 @@ class Index;
 
 namespace noisepage::catalog {
 
+namespace postgres {
+class PgLanguageImpl;
+class PgProcImpl;
+}  // namespace postgres
```

Comment:
Redundant with including the headers above.

Reply:
thanks!